src/App.js

```diff
@@ -1,10 +1,108 @@
+import React from 'react';
+import './App.css';
+import { useState } from 'react';
+import TodoList from './components/TodoList';
+import TodoInsert from './components/TodoInsert';
+
+let nextId = 4;
+
+const App = () => {
+  const [todos, setTodos] = useState([
+    {
+      id: 1,
+      text: 'Todo 1',
+      isDone: false,
+    },
+    {
+      id: 2,
+      text: '할 일 2',
+      isDone: true,
+    },
+    {
+      id: 3,
+      text: 'long input test long input test long input test',
+      isDone: false,
+    },
+  ]);
+  const doneCnt = todos.filter((todo) => todo.isDone).length;
+  const todoCnt = todos.length - doneCnt;
+
+  const onInsertTodo = (text) => {
+    if (text === '') return;
+    const todo = {
+      id: nextId,
+      text,
+      isDone: false,
+    };
+    setTodos((todos) => todos.concat(todo));
+    ++nextId;
+  };
```

Comment:
개인적으로 이렇게 아무것도 리턴하지 않는 리턴문을 작성하는 것보다는 아래와같이 작성하는 게 더 좋아 보입니다!

```suggestion
const onInsertTodo = (text) => {
    if (text) {
      const todo = {
        id: nextId,
        text,
        isDone: false,
      };
      setTodos((todos) => todos.concat(todo));
      ++nextId;
    }
  };
```

Reply:
원래 text === '' 인 경우 alert 처리를 했었는데 UI를 수정하면서 alert 코드만 삭제한채로 두어서 코드가 지저분해진것 같습니다. 말씀하신대로 작성하는것이 더 좋아 보이네요! 많이 반복되는 패턴인데 다음부터는 간결하게 작성할 수 있도록 하겠습니다!